package.json

```diff
@@ -65,6 +65,7 @@
     "react-i18next": "12.2.0",
     "react-refresh": "0.10.0",
     "tailwindcss": "3.2.7",
+    "tailwindcss-rtl": "^0.9.0",
```

Comment:
I'm in total agreement about the ease of these new utilities. However, given that we've already run into a limitation of this unmaintained library, what do you think about using the code from the library directly in our tailwind config? Since we are going to be heavily using direction utilities, maybe we should have control over them. [Here](https://github.com/20lives/tailwindcss-rtl/blob/master/src/marginUtilities.js) is all that it would take to add margin utilities, including negative margins. We wouldn't need to add everything the library supports to start, just the ones that we need.

Reply:
I didn't even think of this. Good idea.